GitUI/UserControls/FileStatusList.cs

```diff
@@ -1307,6 +1307,12 @@ private void FilterComboBox_LostFocus(object sender, EventArgs e)
         private Regex _filter;
 
         #endregion Filtering
```

Comment:
Nitty picky: The method should be in the region

Reply:
Thanks!